### PR TITLE
Isuse #SB-21758 feat: Chatbot API Documentation

### PR DESCRIPTION
**Pull request by JagadishPujari**

### CODE REVIEW

**Comment by keshavprasadms** [comment]
Hello @JagadishPujari,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://643.qa.docs.sunbird.org/3.5.0/ .<br/> **Note:** Only _3.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://643.qa.docs.sunbird.org/3.5.0/apis/chatbotapi// <br /> - http://643.qa.docs.sunbird.org/3.5.0/apis/chatbotapi// <br />  <br/> **Modified sections are -** <br />

File: apis/chatbotapi/chatbotapi.yaml [hunk 1]
```diff
@@ -0,0 +1,219 @@
+openapi: 3.0.1
+info:
+  title: Chat bot API(s)
```
Comment (Sunitashelar) [inline apis/chatbotapi/chatbotapi.yaml:3]: Should be 'Chatbot'

**Review by Sunitashelar — CHANGES_REQUESTED** [review]

**Comment by keshavprasadms** [comment]
Hello @JagadishPujari,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://643.qa.docs.sunbird.org/3.5.0/ .<br/> **Note:** Only _3.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://643.qa.docs.sunbird.org/3.5.0/apis/chatbotapi// <br /> - http://643.qa.docs.sunbird.org/3.5.0/apis/chatbotapi// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @JagadishPujari,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://643.qa.docs.sunbird.org/3.5.0/ .<br/> **Note:** Only _3.5.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://643.qa.docs.sunbird.org/3.5.0/apis/chatbotapi// <br /> - http://643.qa.docs.sunbird.org/3.5.0/apis/chatbotapi// <br />  <br/> **Modified sections are -** <br />

File: apis/chatbotapi/chatbotapi.yaml [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+  termsOfService: https://github.com/project-sunbird/sunbird-commons/blob/master/LICENSE
+  contact:
+    email: info@sunbird.org
+  version: "1.0"
+  
+externalDocs:
+ description: "Testcases"
+ url: "https://www.getpostman.com/collections/aad34d927df640d90727"
+paths:
+  /bot:
```
Comment (SMYALTAMASH) [inline apis/chatbotapi/chatbotapi.yaml:14]: this should be /chat/bot @JagadishPujari , please make sure you add /chat to all the paths below as well.

File: apis/chatbotapi/index.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+---
+<!DOCTYPE html>
+<html>
+	<head>
+		{% include head.html %}
+		<meta name="robots" content="index, follow">
+		<script src="https://rebilly.github.io/ReDoc/releases/v1.19.1/redoc.min.js"> </script>
```
Comment (SMYALTAMASH) [inline apis/chatbotapi/index.html:8]: this is a wrong script URL! check the api-documentation branch contentapis index.html and update accordingly.

**Comment by keshavprasadms** [comment]
Hello @JagadishPujari,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://643.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://643.qa.docs.sunbird.org/api-documentation/apis/chatbotapi// <br /> - http://643.qa.docs.sunbird.org/api-documentation/apis/chatbotapi// <br />  <br/> **Modified sections are -** <br />

**Pull request merged**